pkg/sdk/errors.go

```diff
@@ -8,6 +8,9 @@ import (
 )
 
 var (
+	ErrNilOptions                    = errors.New("options cannot be nil")
```

Comment:
why this is exported? do we need these errors to be referenced outside the SDK package?

Reply:
No, I'll make them private